book/website/communication/citable/citable-linking.md

```diff
@@ -62,6 +66,440 @@ Whichever route you decide to take, it is important to double-check the proof of
+- OSF's native connections between project components are automatic
+:::
+:::{tab-item} Dryad
+:sync: tab4
+- During submission, there's a "Related Works" section
+- Add manuscript DOIs automatically if submitting through journal integration
+- Add additional related resources manually with relationship types
+:::
+::::
+
+
+**For institutional repositories:**
+- Check your repository's documentation or contact repository staff
+- They can add related identifier metadata even if there's no self-service option
+
+### Best Practices for Linking Outputs
+
+- **Link early**: Add relationships when you first deposit outputs, or update metadata as soon as related outputs are published
+- **Link bidirectionally**: If a dataset supplements a paper, add the link from both sides if possible
+- **Be specific**: Use the most precise relationship type (IsSupplementTo is more specific than References)
+- **Link comprehensively**: Connect all related outputs (data, code, paper, protocols, presentations)
+- **Update when publishing**: If you reserved a DOI for a paper, update data/code metadata once the paper has its final DOI
+
+### Example: Connecting a Complete Research Project
+
+Imagine a research project with multiple outputs:
+- Raw dataset (DOI: 10.1234/raw-data)
+- Cleaned dataset (DOI: 10.1234/clean-data)
+- Analysis code (DOI: 10.5281/zenodo.123456)
+- Preprint (DOI: 10.1101/2024.01.001)
+- Published paper (DOI: 10.1016/journal.2024.001)
+
+Connection metadata might look like:
+
+**Clean dataset metadata** includes:
+- IsDerivedFrom: 10.1234/raw-data
+- IsSupplementTo: 10.1016/journal.2024.001
+- IsCompiledBy: 10.5281/zenodo.123456
+- IsVersionOf: 10.1101/2024.01.001 (if the preprint also describes this data)
+
+**Analysis code metadata** includes:
+- Compiles: 10.1234/clean-data
+- IsSupplementTo: 10.1016/journal.2024.001
+
+**Published paper metadata** includes:
+- IsSupplementedBy: 10.1234/clean-data
+- IsSupplementedBy: 10.5281/zenodo.123456
+
+This creates a rich, queryable network where finding any one output leads to discovering all related materials.
+
+(cm-citable-linking-versions)=
+## Managing Versions Through PIDs
+
+Research outputs often evolve over time: you might fix errors in a dataset, improve code, or update protocols.
+Persistent identifiers provide structured ways to handle versioning while maintaining clear connections between versions.
+
+### Version DOIs vs. Concept DOIs
+
+Some repositories (notably Zenodo) implement a two-level DOI system:
+
+Version DOI
+:
+- Points to one specific version of a resource
+- Never changes - always points to exactly that version
+- Use this when you want to cite a specific version (for reproducibility)
+- Example: `10.5281/zenodo.3332807` (version 1.0)
+
+Concept DOI
+:
+- Points to the resource as a whole, across all versions
+- Always resolves to the latest version
+- Use this when you want to cite the work in general
+- Example: `10.5281/zenodo.3332806` (concept, always latest)
```

Comment:
Do you want these to be definitions? I don't think this syntax will render like you want. It would be better to just write sentences.